dace/frontend/python/newast.py

```diff
@@ -2926,8 +2966,10 @@ def visit_Call(self, node: ast.Call):
             # Remove newly-defined symbols from arguments
             if mapping is not None:
                 symbols -= set(mapping.keys())
-            if len(symbols) > 0:
-                mapping = mapping or {}
+            # if len(symbols) > 0:
+            #     mapping = mapping or {}
+            # TODO: Why above the None fix was applied when there were symbols?
```

Comment:
I don't remember why, but we definitely added a test for that, so if it passes it's fine with me.

Reply:
It is probably somehow related to the `_add_access` changes (for a test I got an error because mapping was None).